web/topogen.sh

```diff
@@ -34,13 +36,25 @@ echo "Extract metadata"
   | "$NODE_MODULES_PATH/ndjson-map" 'd.properties' \
 )> build/world_states_props.json
 
+("$NODE_MODULES_PATH/shp2json" -n build/Canary_Islands.shp \
+  | "$NODE_MODULES_PATH/ndjson-map" 'd.properties' \
+)> build/third_party_props.json
+
+# We could use a country filter here to remove unneeded natural earth shapes.
 # Parse countries
 echo 'Parsing countries..'
 ("$NODE_MODULES_PATH/shp2json" -n build/${COUNTRIES_FILENAME}.shp \
   | "$NODE_MODULES_PATH/ndjson-map" '(d.id = (d.properties.ISO_A3 != "-99") ? d.properties.ISO_A3 : d.properties.ADM0_A3, d.oldProps = d.properties, d.properties = {}, d.properties.subid = d.oldProps.SU_A3, d.properties.code_hasc = d.oldProps.code_hasc, delete d.oldProps, d)' \
-  | "$NODE_MODULES_PATH/ndjson-filter" 'd.id' \
+  | "$NODE_MODULES_PATH/ndjson-filter" "d.id" \
 )> build/tmp.json
 
+# Parse 3rd party
+echo 'Parsing 3rd party..'
+("$NODE_MODULES_PATH/shp2json" -n build/Canary_Islands.shp \
```

Comment:
Seems like you just need the shp, so maybe we can remove the others from git?

Reply:
I think the others are required for the shp file to work right. When we download natural earth shapes they include those extra files.

Reply:
I think the shp file is a self-contained file - not sure though. Would be surprised otherwise.

Reply:
https://en.wikipedia.org/wiki/Shapefile, looks like the other parts are needed. Best to keep them for now imo.

Reply:
👍